src/Login.js

```diff
@@ -1,55 +1,49 @@
-import React, {useState} from 'react';
+import React, { useState } from "react";
```

Comment:
Single Quote Format?

Reply:
I guess I can tell prettier to follow single quote format. I can look into it later. 

All the test cases are under `__tests__` directory. 

https://github.com/anarchaworld/307-Gather-Frontend/pull/5#issuecomment-1455206266